deps/src/main/java/com/microsoft/azure/sdk/iot/deps/util/Base64.java

```diff
@@ -1,324 +0,0 @@
-// Copyright (c) Microsoft. All rights reserved.
```

Comment:
We don't need our own implementation of base64 encoding/decoding. Apache commons will handle that for us